SU-HSDO/suhumsci
# fix:(101,83):  code climate linting

# READY FOR REVIEW

## Summary
Fixes code climate issues we didn't see before.

## Need Review By (Date)
asap

## Urgency
medium

## Steps to Test
1. Check the code climate checks are passing for the PR I have here against develop:https://github.com/SU-HSDO/suhumsci/pull/451

## PR Checklist
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
- [Sparkbox PR Checklist](../docs/SparkboxPRChecklist.md)
